common/changes/@uifabric/migration/keco-rm-DetailsRowCheck-isSelected_2019-04-11-06-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Add code mod for removal of DetailsRowCheck.isSelected in favor of 'selected'",
+      "packageName": "@uifabric/migration",
+      "type": "none"
```

Comment:
Figured _none_ made the most sense for now since this will all be grouped into `7.0.0` I imagine cc: @jdhuntington  @cliffkoh

Reply:
It primarily depends on what @jdhuntington's strategy is but I think none is fine